group_project/assignment_3.py

```diff
@@ -1,5 +1,13 @@
+
 def replace_bad_words(a_string, word_to_replace, new_word):
-    pass
+    find_word = a_string.find(word_to_replace)
+    word_length = len(word_to_replace)
+    replace = find_word + word_length
+    if find_word != -1:
+        a_string = a_string.replace(word_to_replace, new_word)
+        print (a_string)
+        return (a_string)
+    return a_string
 
```

Comment:
This is a good solution, but I think it could have been shorter with the replace() method.